lltodk.ml

```diff
@@ -1064,7 +1063,7 @@ let output_term oc phrases ppphrases llp =
   let dkproof = make_proof_term (List.hd goal) prooftree in
   fprintf oc "require open Logic.Zenon.FOL Logic.Zenon.LL Logic.Zenon.ND Logic.Zenon.ND_eps Logic.Zenon.ND_eps_full Logic.Zenon.ND_eps_aux Logic.Zenon.LL_ND;\n";
   fprintf oc "require open Logic.Zenon.zen;\n";
-  if !Globals.signature_name <> "" then fprintf oc "require %s.%s as %s;\n" !Globals.signature_name !Globals.signature_name !Globals.signature_name;
+  if !Globals.signature_name <> "" then fprintf oc "require %s as S;\n" !Globals.signature_name;
```

Comment:
I think you still need "require %s.%s as S" for it to work, otherwise lambdapi complains  about an unqualified quantifier.

Reply:
But Globals.signature_name can be a qualified name, can't it?

Reply:
Indeed.